### DISCUSSION TITLE
Small directional ping buff

### DISCUSSION THREAD

**Discussion by LePiromano** [FakeFishGames/Barotrauma#6719] [Ideas and suggestions]
There's another very upvoted suggestion that overhauls the sonar terminal completely, this one is just a temporary  "fix" for a completely useless feature:

Simple: Make the directional ping increase the sonar's view distance on the direction that you're aiming with it, at the cost of losing visibility on the other sides of the sonar.

**Comment by SpazzMaticus107** [comment]
Me and my bunch always talk about how unintuitive the directional sonar is. In fact all of us imagined it working just as you're suggesting. Not to mention when I first toggled directional mode I thought to myself I was bugged since the sonar doesn't zoom out for me to see more. Those changes would be very welcome for new and old players alike.

**Comment by Ek-Videogames** [comment]
Last we heard the directional sonar still triggers enemies outside of it's actual FOV? but stealth non-existence is its own discussion entirely.

**Comment by iridescent-lightning** [comment]
This has been fixed by mod Barotrauma Die Hard. All features you want from directional sonar are there.

**Comment by G-M-twostay** [comment]
I heard that directional ping makes u completely deaf to other directions, is this true? If yes then making u able to hear other directions would be great.
